lib/elixir/test/elixir/fixtures/parallel_struct/bar.ex

```diff
@@ -1,3 +1,3 @@
-defmodule Bar do
+defmodule :bar do
```

Comment:
I am not a fan of changing this test which is meant to check for deadlocks. Maybe we can add a test to map_test.exs that looks like this:

```elixir
defmodule :elixir_struct_from_erlang_module do
  defstruct :hello
  def world(%:elixir_struct_from_erlang_module{} = struct), do: struct
end
```

?